tools/editor/property_editor.cpp

```diff
@@ -1864,6 +1867,15 @@ void CustomPropertyEditor::_modified(String p_string) {
 	updating=false;
 }
 
+void CustomPropertyEditor::_emit_changed_whole_or_field() {
+
+	if (!Input::get_singleton()->is_key_pressed(KEY_SHIFT)) {
```

Comment:
Making this configurable would be great, but in my tests with other keys, for instance, Control+Enter was ignored. Anyway, if you can point me on it could be done, I'll try.